docs/data/sdks/typescript-browser/index.md

```diff
@@ -163,6 +164,18 @@ amplitude.init(API_KEY, OPTIONAL_USER_ID, {
 });
 ```
 
+The following are the event info got tracked in the page view events.
+
+|<div class="big-column">Name</div>| Description| Default Value|
+|---|----|---|
+|`event_type`| `string`. The event type for page view event. Configurable through `defaultTracking.pageViews.eventType` or enrichment plugin. | `[Amplitude] Page Viewed`, `[Amplitude] Page View` for version previous 1.9.1 |
```

Comment:
> [Amplitude] Page View` for version previous 1.9.1

We don't have page view tracking under default tracking, so I think better to remove this

Reply:
https://github.com/amplitude/Amplitude-TypeScript/blob/5590f7d77d93362d2f3db6d47116ca502afd1fc4/packages/analytics-browser/src/browser-client.ts#LL121C41-L121C64

v1.9.0 do have defaultTracking.

Reply:
- v1.8.0 is using 'Page View' as the event type. Check [here](https://github.com/amplitude/Amplitude-TypeScript/blob/e9b73ad53882d499ca14e3d53e8885825554d0d9/packages/analytics-browser/src/browser-client.ts#L87) and [here](https://github.com/amplitude/Amplitude-TypeScript/blob/e9b73ad53882d499ca14e3d53e8885825554d0d9/packages/analytics-client-common/src/attribution/campaign-tracker.ts#L84-L91) if not using pageview plugin. 
- If we use the web attribution plugins, the page view event will be [this](https://github.com/amplitude/Amplitude-TypeScript/blob/e9b73ad53882d499ca14e3d53e8885825554d0d9/packages/plugin-page-view-tracking-browser/src/page-view-tracking.ts#L47),  and [this is the event properties.](https://github.com/amplitude/Amplitude-TypeScript/blob/e9b73ad53882d499ca14e3d53e8885825554d0d9/packages/plugin-page-view-tracking-browser/src/page-view-tracking.ts#L109-L118) 
- for the version between v1.9.0 - 1.9. the page view event type will be  `[Amplitude] Page View`. Check [here](https://github.com/amplitude/Amplitude-TypeScript/blob/5590f7d77d93362d2f3db6d47116ca502afd1fc4/packages/analytics-browser/src/browser-client.ts#LL121C41-L121C64). 
Due to the complicity of the event page view in browser SDK. Should we always only mention the latest version in this doc? 
wdyt? @justin-fiedler @Mercy811

Reply:
v1.9.0 is considered a bad release so I'll lean on not putting this in a doc since we won't recommend it anyway. The default events tracked there are not planned in Data too.

Reply:
For v1.8.x and below, page views are tracked under `attribution` and not under `defaultTracking`. My concern was that this section was for configuration under `defaultTracking` and so mentioning here seems misleading

Reply:
I rephrase this to `[Amplitude] Page Viewed` from version 1.9.1.

Reply:
I think we should focus on the latest functionality. What about adding annotation to `Page Views` that explains the feature used to be on `attribution` and had different names for the events?

<img width="776" alt="image" src="https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/assets/5790872/f3f083a4-dd8f-4bb1-a74f-e470aad684e5">